### ISSUE TITLE
--output-resolve-crossrefs fails with 'Can't call method "set_field" on an undefined value' when a missing crossref is used twice

### ISSUE THREAD

**Issue by logological** [plk/biber#271]
**Summary:**

When running in tool mode with the `--output-resolve-crossrefs` option, Biber crashes the second time it encounters a given `crossref` value with no corresponding citation key in the resource.

**Steps to reproduce:**

1. Create a `.bib` file with two entries; each entry should include a `crossref` field whose value is the same non-existent citation key.  Example:
```
@book{foo,
  author       = {Foo},
  title        = {Foo},
  year         = 2019,
  crossref     = {baz},
}

@book{bar,
  author       = {Bar},
  title        = {Bar},
  year         = 2019,
  crossref     = {baz},
}
```
2. Run Biber in tool mode to resolve the crossreferences:
```
biber --tool test.bib --output-resolve-crossrefs
```

**Observed result:**

3. Biber crashes with the following error message:
```
Can't call method "set_field" on an undefined value at /tmp/par-707379/cache-d31705f85102d579faeed3daf7f26b3d37aae75b/inc/lib/Biber.pm line 1438.
```

**Expected result:**

3. Biber should have simply output a warning that the citation key could not be found and continued to process the file, ignoring the invalid `crossref`.

**Additional notes:**

1. I can reproduce this problem only in tool mode.  If I use such a bibliography file in a LaTeX document and then use Biber to process it, it does not crash but rather correctly emits a warning for each entry with an invalid `crossref` value.
2. The problem occurs only when the two `crossref` keys have the same invalid value.  Multiple entries with invalid `crossref` values are fine as long as the values are distinct.
3. This problem may be related to an issue I previously reported (#115), which had a similar error message.

**Comment by plk** [comment]
Thank you for reporting this - it was a bug and is fixed in 2.13 DEV version.

**Comment by plk** [comment]
Fixed in latest 2.13 release.
